### DISCUSSION TITLE
스터디 2기 2주차 모임

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#343] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

세 번째 모임도 전 모임과 같이 지원자를 받고 각각 문제에 대해서 설명 및 논의하는 시간을 가지겠습니다!

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을 one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다!(https://discord.gg/BJdHgZxg?event=1274830883009859624)

**Comment by TonyKim9401** [comment]
Construct Binary Tree From Preorder And Inorder Traversal / TonyKim9401

**Reply by obzva** [reply]
[iterative한 풀이](https://github.com/DaleStudy/leetcode-study/pull/346#discussion_r1725946966)도 짧게 소개드리겠습니다!

**Reply by TonyKim9401** [reply]
발표때 사용한 PPT 올려둡니다!
https://docs.google.com/presentation/d/1PbgvawXD2zf_HhFH00IleZ6CbvvNbaXh9__SyLrs2MA/edit?hl=ko#slide=id.g2f5c1cfffde_0_51

**Comment by EgonD3V** [comment]
[Encode and Decode Strings](https://github.com/DaleStudy/leetcode-study/issues/238) / EGON

**Comment by jdalma** [comment]
Decode Ways / 정현준
